src/api/app/assets/stylesheets/webui/responsive_ux/grid.scss

```diff
@@ -27,31 +27,22 @@
 
 @include media-breakpoint-up(xl) {
   #grid {
-    grid-template-columns: $left-navigation-width auto;
-    grid-template-rows: $top-navigation-height auto;
+    grid-template-columns: min-content auto;
+    grid-template-rows: $top-navigation-height 1fr auto;
     grid-template-areas:
       'top-navigation top-navigation'
-      'left-navigation content';
+      'left-navigation content'
+      'toggle-sidebar-button content';
 
     #left-navigation-area {
       grid-area: left-navigation;
       display: block;
+      grid-template-columns: min-content auto;
```

Comment:
This rule should be removed since it doesn't have any effect because this element is `display: block`.

Reply:
Right. And also this rule is specified for `#grid` already. Seems like I added it mistakenly here. Removed.